bitcoin-extensions/styles.css

```diff
@@ -0,0 +1,22 @@
+html {
```

Comment:
Generally speaking, it is best practice to use the * selector when changing universal styles like the background color or text color of an entire page. 

* {
// some css here
}

Reply:
make sure to use the star without the quotes. Weird formatting thing in github turns it to a dot if i leave off the quotation marks.